### PR TITLE
misc: salt remove authorizedAddress (device addr)

### PR DESCRIPTION
**Pull request by kbehouse**
Asana Task: [Task](https://app.asana.com/0/1200122517292716/1203647032236620)
salt remove authorizedAddress (device address)

### CODE REVIEW

File: truffle-config.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+require("dotenv").config();
```
Comment (popodidi) [inline truffle-config.js:1]: i think we have a `truffle.js`?

Comment (kbehouse) [reply truffle-config.js:1]: already done

File: truffle-config.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+
+const CONSISTENT_DEPOLY_ADDR = "0xB3c12C9535626cfFef5FE76d5A4e198A5e753FD9";
+// const CONSISTENT_DEPOLY_ADDR = "0x70ebDF2a1C9C7A45141760419c42056E3DDfA690";
+// const CONSISTENT_DEPOLY_ADDR ="0x1Dde3A77975235E6DdB31389c3D599dE67AbA1Ad";
```
Comment (popodidi) [inline truffle-config.js:6]: what are these addresses? remove or add comment for the purpose, and remove if they are irrelevant

File: truffle-config.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    ethereum: {
+      provider: new HDWalletProvider(
+        process.env.SAME_ADDR_KEY,
+        process.env.ETHEREUM_URL
+      ),
+      network_id: 1,
+      url: process.env.ETHEREUM_URL,
+      from: CONSISTENT_DEPOLY_ADDR,
+      gas: 4000000, // 8M
+      gasPrice: 29800000000, // 37.6 gwei
```
Comment (popodidi) [inline truffle-config.js:21]: inconsistent between code and comment

File: truffle-config.js [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+// const CONSISTENT_DEPOLY_ADDR = "0x70ebDF2a1C9C7A45141760419c42056E3DDfA690";
+// const CONSISTENT_DEPOLY_ADDR ="0x1Dde3A77975235E6DdB31389c3D599dE67AbA1Ad";
+module.exports = {
+  // See <http://truffleframework.com/docs/advanced/configuration>
+  // to customize your Truffle configuration!
+  networks: {
+    // mainnet config
+    ethereum: {
+      provider: new HDWalletProvider(
+        process.env.SAME_ADDR_KEY,
```
Comment (popodidi) [inline truffle-config.js:14]: nit: use the same
```
const {.....} = process.env;
```
to have an overview of the supported environment variables.

Comment (kbehouse) [reply truffle-config.js]: thanks, using 
const {SAME_ADDR_KEY, SAME_ADDR, ETHEREUM_URL, SEPOLIA_URL,
      ETHERSCAN_API_KEY, POLYGONSCAN_API_KEY, BSCSCAN_API_KEY, SNOWTRACE_API_KEY} = process.env;

**Review by popodidi — APPROVED** [review]

**Pull request merged**